gpuselect/Makefile

```diff
@@ -1,10 +1,13 @@
+PYV=$(shell python -c 'import sys; print("%d%d" % (sys.version_info[:2]))')
+PYVDOT=$(shell python -c 'import sys; print("%d.%d" % (sys.version_info[:2]))')
+CUDA_ROOT ?= "/usr/local/cuda"
 all: _gpuselect.so
 
 gpuselect.o: gpuselect.cpp
-	/usr/local/cuda-7.5/bin/nvcc -Xcompiler -fPIC -c -I/usr/local/cuda/include -I/usr/include/python2.7 -o gpuselect.o gpuselect.cpp
+	${CUDA_ROOT}/bin/nvcc -Xcompiler -fPIC -c -I${CUDA_ROOT}/include -I/usr/include/python${PYVDOT} -o gpuselect.o gpuselect.cpp
```

Comment:
This depends on CUDA_ROOT containing bin which I'm not yet convinced it should. Let's see what comes out of our discussion on Teams